autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterV2Controller.java

```diff
@@ -17,7 +18,8 @@
 /**
  * @deprecated Database ID based endpoints are deprecated for removal.
  */
-@Component
+@Controller
+@DisableCheckPermissions
```

Comment:
I think this should be changed to `@InternalOnly` to ensure that noone will call this

Reply:
ok